gateway-messages/src/sp_to_mgs.rs

```diff
@@ -149,13 +149,42 @@ pub struct SpState {
 #[derive(
     Debug, Clone, Copy, PartialEq, Eq, SerializedSize, Serialize, Deserialize,
 )]
-pub struct RotState {
+pub enum RotSlot {
+    A = 0,
```

Comment:
What do we gain from the explicit `A = 0`, `B = 1`?

Reply:
Nothing! I will remove it.